src/eq_schema/schema/QuestionnaireFlow/index.js

```diff
@@ -0,0 +1,32 @@
+const { filter } = require("lodash");
+class QuestionnaireFlow {
+  constructor(questionnaireJson) {
+    this.type = questionnaireJson.hub ? "Hub" : "Linear";
+    this.options = this.build_options(questionnaireJson);
+  }
+
+  build_options(questionnaireJson) {
+    const options = {}
+    if(questionnaireJson.hub) {
+      const required_completed_sections = this.build_required_sections(questionnaireJson);
+      if(required_completed_sections.length) {
+        options.required_completed_sections = required_completed_sections;
+      }
+      return options;
+    };
+    if(questionnaireJson.summary) {
+        options.summary = {
+          collapsible: questionnaireJson.collapsibleSummary
+      };
+    };
+    return options;
+  }
+
+  build_required_sections(questionnaireJson){
+    const sections = filter(questionnaireJson.sections, {"required": true});
+    const required_sections = sections.map((section) => section.id);
+    return required_sections;
+  }
+}
+
+module.exports = QuestionnaireFlow;
```

Comment:
You've still got some snake case in here that should be camel case.

For instance,

- Variable names, such as `const required_sections` and `const required_completed_sections`
- Functions, such as `build_options` and `build_required_sections`